This patch makes a change so that UQM will find zlib. You may need to apply this
to the UQM source tree to make it build:

patch -p0 < ~/uqm-win-alterations.patch

diff -U 3 -H -d -r -N -- uqm-0.7.0/build/unix/config_proginfo_host uqm-win/uqm-0.7.0/build/unix/config_proginfo_host
--- build/unix/config_proginfo_host	2011-05-20 15:25:22.000000000 -0400
+++ build/unix/config_proginfo_host	2012-10-31 16:07:41.000000000 -0400
@@ -260,7 +260,7 @@
 LIB_zlib_CFLAGS=""
 case "$HOST_SYSTEM" in
 	MINGW32*|CYGWIN*|cegcc)
-		LIB_zlib_LDFLAGS="-lzdll"
+		LIB_zlib_LDFLAGS="-lz"
 		;;
 	ARMV5|WINSCW|GCCE)
 		LIB_zlib_LDFLAGS=""

